spec/javascripts/models/beta/sprint_spec.js

```diff
@@ -0,0 +1,66 @@
+import * as Sprint from 'models/beta/sprint';
+
+describe('Sprint model', () => {
+  const doneSprint = { hasStories: '' }
+  const undoneSprint = {}
+
+  describe('sort', () => {
+    describe('when is done sprints', () => {
+      const amountOfSprints = [1,10,100]
+
+      amountOfSprints.forEach(amount => {
+        describe(`when have ${amount} sprints`, () => {
+          let doneSprints = Array(amount).fill(doneSprint);
+          let changedSprints;
+
+          beforeEach(() => {
+            changedSprints = Sprint.sort(doneSprints);
+          });
+
+          it('return reverse order', () => {
+            expect(changedSprints).toEqual(doneSprints.reverse());
+          });
+    
+          it(`return ${doneSprints.length} sprints`, () => {
```

Comment:
Since this is a sort method and will not change the size of an array then I wouldn't test it.